### PR TITLE
add htmlEntryPoint option to transformer

### LINKED ISSUE #10: Add transformer option to rewrite original entry point script tag
Should be called something like `htmlEntryPoint` and take a file path to an html file. It will then search for script tags linking to `entryPoint` and change them to `newEntryPoint`.

### PR DESCRIPTION
**Pull request by jakemac53**
fixes https://github.com/dart-lang/initialize/issues/10

### CODE REVIEW

File: lib/transformer.dart [hunk 1]
```diff
@@ -65,12 +69,23 @@ class InitializeTransformer extends Transformer {
     if (newEntryPoint == null) {
       newEntryPoint = entryPoint.replaceFirst('.dart', '.bootstrap.dart');
     }
-    return new InitializeTransformer(entryPoint, newEntryPoint);
+    var htmlEntryPoint = settings.configuration['htmlEntryPoint'];
```
Comment (sigmundch) [inline lib/transformer.dart]: we could do the change from `'htmlEntryPoint'` -> `'html_entry_point'` right away or separately (issue #14)

File: lib/transformer.dart [hunk 2]
```diff
@@ -85,6 +100,47 @@ class InitializeTransformer extends Transformer {
       }
     });
   }
+
+  Future _replaceEntryWithBootstrap(Transform transform) {
+    // For now at least, _htmlEntryPoint, _entryPoint, and _newEntryPoint need
```
Comment (sigmundch) [inline lib/transformer.dart]: Would it be enough if entrypoint/newentrypoint are not in lib? Then we can just use a relative path.

**Comment by sigmundch** [comment]
lgtm

File: lib/transformer.dart (hunk 2 above)
Comment (jakemac53) [inline lib/transformer.dart]: Ya, we could just not allow entry points in lib at all, I think I will do that.

File: lib/transformer.dart (hunk 1 above)
Comment (jakemac53) [inline lib/transformer.dart]: I think we can update all of these at once later

File: lib/transformer.dart (hunk 2 above)
Comment (jakemac53) [inline lib/transformer.dart]: On second thought I don't think we really want people reaching into web from test, etc.

**Comment by jakemac53** [comment]
submitting for now, we can revisit the allowing entry points in different folders thing in a later cl if its needed

**Pull request merged**